src/pages/iou/request/step/IOURequestStepParticipants.js

```diff
@@ -91,7 +112,7 @@ function IOURequestStepParticipants({
         >
             <MoneyRequestParticipantsSelector
                 ref={(el) => (optionsSelectorRef.current = el)}
-                participants={participants}
+                participants={isSplitRequest ? participants : []}
```

Comment:
One of the reasons I added back `isSplitRequest ? participants : []` is for this case:
1. User adds 2 participants to split
2. User changes the URL from /split to /request
3. If we don't use that logic above, they will see 2 users selected with a title Manual because the `iouType` is request

And because we added this logic back, I think removing the `IOU.setMoneyRequestParticipants_temporaryForRefactor(transactionID, []);` would help reducing unnecessary operation (especially it removes the participant from the onyx while going back which would add more lag to the app).

Reply:
So, do we want to remove it? Or just leave it?